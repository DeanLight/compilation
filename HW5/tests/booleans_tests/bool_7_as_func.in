bool funBool(bool var){
if (var) print("var is true :)\n");
else print("var is false :(\n");

}

void main()
{
	bool t = true;
	bool f = false;
	funBool(f);
}