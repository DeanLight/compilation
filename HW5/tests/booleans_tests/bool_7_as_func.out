Loaded: ./exceptions.s
var is false :(
